pkg/router/controller/status_test.go

```diff
@@ -896,3 +896,7 @@ func (i *fakeInformer) HasSynced() bool {
 func (i *fakeInformer) LastSyncResourceVersion() string {
 	panic("not implemented")
 }
+
+func (i *fakeInformer) SetTransform(handler cache.TransformFunc) error {
```

Comment:
And this change as well - why it is required for the bump?

Reply:
https://github.com/kubernetes/client-go/commit/54928eef9f824667b23a938188498992d437156a